include/libp2p/protocol/kademlia/impl/peer_routing_table_impl.hpp

```diff
@@ -91,17 +142,25 @@ namespace libp2p::protocol::kademlia {
 
     Bucket split(size_t commonLenPrefix, const NodeId &target) {
       Bucket b{};
-      // remove shifts all elements "to be removed" to the end, other elements
-      // preserve their relative order
-      auto new_end = std::remove_if(begin(), end(), [&](const auto &bpi) {
-        return bpi.node_id.commonPrefixLen(target) > commonLenPrefix;
-      });
 
-      b.assign(std::make_move_iterator(new_end),
-               std::make_move_iterator(end()));
+      std::list<BucketPeerInfo> newPeers;
```

Comment:
Google C++ style [guide](https://google.github.io/styleguide/cppguide.html#Variable_Names) prescribes to use snake case for names of that kind variables. (e.g. ```new_peers, common_len_prefix```).